setup/templates/steps/config.html

```diff
@@ -48,10 +48,18 @@
 </div>
 
 <div class="form-group">
-  <label>Authentication rate limit (per source IP address)</label>
+  <label>Authentication rate limit per IP for failed login attempts for non-existing accounts</label>
```

Comment:
There's one f too many here: for failed login attempts *or* non-existing accounts